tests/BuildScriptGenerator.Common.Tests/VersionInfoTest.cs

```diff
@@ -0,0 +1,60 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System;
+using Xunit;
+using Microsoft.Oryx.Tests.Common;
+
+namespace Microsoft.Oryx.BuildScriptGenerator.Common.Tests
+{
+    public class VersionInfoTest : IClassFixture<TestTempDirTestFixture>
+    {
+        public VersionInfoTest(TestTempDirTestFixture testFixture)
+        {
+        }
+
+        [Theory]
+        [InlineData("3.9.0b1", "3.9.0-b1")]
```

Comment:
Should not matter, but try adding a case for PHP style preview versions too

Reply:
yeah got it, forgot to add PHP preview versions, will add it